packages/german-words/0.0.3/_manifest.yml

```diff
@@ -0,0 +1,7 @@
+name: "german-words"
+title: "German Wörter"
+description: "A Package that includes German words."
+author: "Fatih Simsek"
+version: 0.0.3
+homepage: "https://github.com/Fatih5252/German-Words"
+tags: ["German", "Germany", "Language", "Learn", "languages"]
```

Comment:
I don't think "Language" adds anything to "Languages", or "Germany" to "German", so would remove the former in each case, but there are a few others you might consider adding, like "translation" and "dictionary".

It's helpful for users browsing the hub if packages share existing tags, rather than add unique ones, except in particular circumstances.